app/Community/Enums/ActivityType.php

```diff
@@ -27,8 +27,6 @@ abstract class ActivityType
 
     public const ClosedTicket = 10;
 
-    public const BeatGame = 11;
```

Comment:
There are 70k entries in the Activity table with ActivityType=11. Do those need to be deleted, or should this enum value be kept?

Reply:
I am not sure what the correct strategy here is. If we're planning on dropping the whole table eventually anyway, it may not be a big deal to keep these around.

Reply:
@luchaos - thoughts? Keep the enum or delete the records?

Reply:
let's keep the entries and enum for now - this way we can technically reconstruct the last login dates by looking up BeatGame activities.

Reply:
Value restored in https://github.com/RetroAchievements/RAWeb/pull/1797/commits/dbc409eb429380cb63da081b058bf3f72762959d.